code/controllers/subsystem/ticker.dm

```diff
@@ -451,6 +451,8 @@ var/datum/subsystem/ticker/ticker
 			dellog += "Failures : [SSgarbage.didntgc[path]] \n"
 		world.log << dellog
 
+	//Collects persistence features
+	SSpersistence.CollectData()
```

Comment:
The end of round isn't very clean in terms of calling things, right now you either add something in round end code or you set your atom to processing and check for ticker.current_state == GAME_STATE_FINISHED.

The idea of making this a subsystem is to eventually unite all the persistence <s>memes</s> features under it.